test/run_test_confluent.sh

```diff
@@ -204,6 +204,5 @@ if [ $testError -ne 0 ]; then
     RED='\033[0;31m'
     NC='\033[0m' # No Color
     echo -e "${RED} There is error above this line ${NC}"
-    cat $APACHE_LOG_PATH/kc.log
```

Comment:
Not sure if we need it at all. Printing 10k log lines might be useful in CI but for local development it makes things worse.